### ISSUE TITLE
4 player / controller support in PS1 games via playstation tv - some games work, some not. Any clue?

### ISSUE THREAD

**Issue by pimpsupreme** [TheOfficialFloW/Adrenaline#199]
I want to play some good old party games from the ps1 time.. like Micro Machines, WWF Smackdown, ISS Pro, etc. with 3 of my friends. 

I paired 4 ps4 controllers to the playstation tv - they work in the tv/vita menues -  but in a ps1 game (in adrenaline / psp), there were only two controllers recognised.

- I'm new to Adrenaline (and Henkaku) by the way -

Is there any trick to enable the other ones?
Or is this a hell of a job to implement that thing for a one-guy-wish? :-)

Thanks, best regards!

**Comment by pimpsupreme** [comment]
And of course I have the latest version (6.61 Adrenaline-6 fix) installed. I've tested it with ISS Pro 2, WWF Smackdown 2 and Micro Machines.

Maybe there is a "multitap option" or something else to activate?

The games run like a charm, but I can't select more than two controller. 

Regards

**Comment by pimpsupreme** [comment]
OK - I've checked two other games.

Crash Team Racing works with 4 controllers out of the box!!!
So does Bomberman Party Edition!!
But WWF Attitude only accepts 2 controllers....

Sooooooo, does anyone knows why other games just don't work with 4 controllers ("only" with 2 controllers...) or how I can force the "multitap"?

**Comment by dude22072** [comment]
Looking at you list, it seems the only games that are working with multitap are ones that have an official PS Classic release. It's possible that whatever method you are using to get these non ps classic releases are not marking them as multitap compatible.

**Comment by pimpsupreme** [comment]
Hi dude22072, this is a really really good point! Honestly, I made the "eboots" by myself. I have all my old ps1 games in the basement (50+) and thought, this is a good opportunity to make a little "retro party" without trying to get my old ps1 on my new HD-TV....(I've chosen my 10 favorite ps1 multiplayer games)

So it seems, that the emulator itself has a whitelist (?) to enable the multitab / 4-player-mode for some games.

I've tried Hot Shots Golf 2 (Everybodys Golf). That's a ps1 classic too. And yes, it works with 4 players! So you are right.

It seems only working for games that marked as "playstation classics".
